examples/basic/example_08_lsdyna_taylor_bar.py

```diff
@@ -0,0 +1,324 @@
+print(f"Downloaded the geometry file to: {geometry_path}")
+
+###############################################################################
+# Launch Mechanical
+# ~~~~~~~~~~~~~~~~~
+# Launch a new Mechanical session in batch, setting ``cleanup_on_exit`` to
+# ``False``. To close this Mechanical session when finished, this example
+# must call  the ``mechanical.exit()`` method.
+
+mechanical = launch_mechanical(batch=True, cleanup_on_exit=False)
+print(mechanical)
+
+###############################################################################
+# Initialize variable for workflow
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set the ``part_file_path`` variable on the server for later use.
+# Make this variable compatible for Windows, Linux, and Docker containers.
+
+project_directory = mechanical.project_directory
+print(f"project directory = {project_directory}")
+project_directory = project_directory.replace("\\", "\\\\")
+mechanical.run_python_script(f"project_directory='{project_directory}'")
+
+# Upload the file to the project directory.
+mechanical.upload(file_name=geometry_path, file_location_destination=project_directory)
+
+# Build the path relative to project directory.
+base_name = os.path.basename(geometry_path)
+combined_path = os.path.join(project_directory, base_name)
+part_file_path = combined_path.replace("\\", "\\\\")
+mechanical.run_python_script(f"part_file_path='{part_file_path}'")
+
+###############################################################################
+# Download required material files
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Download the required file. Print the file path for the material file.
+
+mat_st_path = download_file("example_08_Taylor_Bar_Mat.xml", "pymechanical", "00_basic")
+print(f"Downloaded the material file to: {mat_st_path}")
+
+# Upload the file to the project directory.
+mechanical.upload(file_name=mat_st_path, file_location_destination=project_directory)
+
+# Build the path relative to project directory.
+base_name = os.path.basename(mat_st_path)
+combined_path = os.path.join(project_directory, base_name)
+mat_file_path = combined_path.replace("\\", "\\\\")
+mechanical.run_python_script(f"mat_file_path='{mat_file_path}'")
+
+# Verify the path
+result = mechanical.run_python_script("part_file_path")
+print(f"part_file_path on server: {result}")
+
+mech_act_code = """
+import os
+import json
+
+# Import Taylor bar geometry
+geometry_import_group = Model.GeometryImportGroup
+geometry_import = geometry_import_group.AddGeometryImport()
+
+geometry_import_format = Ansys.Mechanical.DataModel.Enums.GeometryImportPreference.Format.Automatic
+geometry_import.Import(part_file_path, geometry_import_format, None)
+
+Model.AddLSDynaAnalysis()
+analysis = Model.Analyses[0]
+
+ExtAPI.Application.ActiveUnitSystem = MechanicalUnitSystem.StandardNMMton
+ExtAPI.Application.ActiveAngleUnit = AngleUnitType.Radian
+
+MAT = ExtAPI.DataModel.Project.Model.Materials
+#MAT.Import(mat_file_path)
```

Comment:
@dipinknair can you uncomment MAT.Import(mat_file_path)

Reply:
Yes #171